src/main/java/com/noticemedan/map/model/KDTree/Forest.java

```diff
@@ -1,17 +1,27 @@
 package com.noticemedan.map.model.KDTree;
 
 import com.noticemedan.map.model.MapObject;
-import lombok.Getter;
-import lombok.Setter;
-import java.util.List;
+import lombok.AllArgsConstructor;
 
+import java.util.ArrayList;
+import java.util.List;
 
+@AllArgsConstructor
 public class Forest implements ForestInterface {
-	@Setter private KDTree tree;
+	private KDTree trees[];
 
 	@Override
+	public List<MapObject> rangeSearch(Rect query, int zoomLevel) {
+		ArrayList searchResults = new ArrayList<>();
+		for (int i = 0; i < zoomLevel+1; i++) {
```

Comment:
Hvorfor er zoomLevel +1 ?

Reply:
Det er nemmere at forklare for dig når vi mødes:)